api-report/driver-utils.api.md

```diff
@@ -116,14 +116,17 @@ export function combineAppAndProtocolSummary(appSummary: ISummaryTree, protocolS
 export function configurableUrlResolver(resolversList: IUrlResolver[], request: IRequest): Promise<IResolvedUrl | undefined>;
 
 // @public (undocumented)
-export function createGenericNetworkError(fluidErrorCode: string, message: string | undefined, canRetry: boolean, retryAfterMs?: number, props?: ITelemetryProperties): ThrottlingError | GenericNetworkError;
+export function createGenericNetworkError(fluidErrorCode: string, message: string | undefined, retryInfo: {
+    canRetry: boolean;
+    retryAfterMs?: number;
```

Comment:
@anthony-murphy - How do I think through if this breaking change is a risk in terms of backwards compatibility?  My assumption is that if a package has a build dependency on another package (like driver-utils), then at runtime it will have the code it's built against.  You only have a compat boundary at runtime if you're building against interfaces (i.e. depending on definitions packages).

So, I can feel free to make this breaking change to switch to `retryInfo`, right?  And I have another breaking change coming next (basically reverting PR #7788, which didn't seem to break anything itself).

cc @jatgarg